Automattic/wp-calypso
# Assembler - Remove All category and hide Text, Podcast, and Portfolio

Related to #76587

## Proposed Changes

* Remove All category
* Hide Text, Podcast, and Portfolio
* Open `Blog Posts` category initially
* Remove unused code related to All category from ETK plugin

|BEFORE|AFTER|
|--|--|
|<img width="349" alt="Screenshot 2566-08-25 at 13 05 00" src="https://github.com/Automattic/wp-calypso/assets/1881481/23a8be05-9f2a-4a16-87dc-ab91b27652d8"><br><img width="348" alt="Screenshot 2566-08-25 at 13 06 26" src="https://github.com/Automattic/wp-calypso/assets/1881481/d20a37b1-a406-411c-a377-51a56bad93bf">|<img width="350" alt="Screenshot 2566-08-25 at 13 03 55" src="https://github.com/Automattic/wp-calypso/assets/1881481/ac2f912a-5132-4018-a302-8fb3d6aa63e5">|

## Testing Instructions



* Access the assembler `/setup/with-theme-assembler/patternAssembler?siteSlug={ SITE }`
* Click `Sections` and verify the first category `Blog Posts` opens initially
* Verify that you don't see the categories `Text`, Podcast, and Portfolio` 

## Pre-merge Checklist
